src/Covid19Dashboard/Covid19Dashboard.less

```diff
@@ -77,7 +77,30 @@
 .map-chart {
   width: 100%;
   flex-basis: 200%;
-  max-width: 1080px;
+  //max-width: 1080px; set for limiting world map to one set
+  height: 55vw;
```

Comment:
I don't have any context for how this is implemented so feel free to ignore me 😄, but is this supposed to be `vh`, or is the height relative to the width? I wonder if this is a case where a grid based layout would help?

Reply:
it is supposed to be vw, it is used it to keep the same relative aspect ratio for the map as the screen size changes. I dont think grid would be appropriate as i am not stacking elements, only have one row and need them to flex that is why i used flex box